### PR TITLE
{math}[GCC/11.3.0] nauty v2.8.6

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/n/nauty/nauty-2.8.6-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# prepend "lib" to library files to standarize their name
+postinstallcmds = ["cd %%(installdir)s/lib/ && mv %s lib%s" % (l, l) for l in local_libs]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in local_executables] +
+             ['include/%%(name)s/%s' % h for h in local_headers] +
+             ['lib/lib%s' % l for l in local_libs],
+    'dirs': ['']
+}
```
Comment (boegel) [inline easybuild/easyconfigs/n/nauty/nauty-2.8.6-GCC-11.3.0.eb:49]:
```suggestion
}

sanity_check_commands = ["dreadnaut --help"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3120.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/de8244c0612dc184605c1c41cbf9f535 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**